src/sklearn_evaluation/plot/classification.py

```diff
@@ -131,12 +131,15 @@ def _confusion_matrix(y_true, y_pred, normalize):
     return cm
 
 
-def confusion_matrix(y_true,
-                     y_pred,
-                     target_names=None,
-                     normalize=False,
-                     cmap=None,
-                     ax=None):
+@SKLearnEvaluationLogger.log(feature='plot')
+def confusion_matrix(
+        y_true,
+        y_pred,
+        target_names=None,
+        normalize=False,
+        cmap=None,
+        ax=None,
+        **kwargs):
```

Comment:
I see you added `**kwargs` to many functions and I'm guessing this has to do with the `is_report` flag that you're passing.

this `**kwargs` will create problems because if a user has a typo in any of the function parameters, the function won't throw an error, so I'd say let's remove them. I'm guessing the only drawback would be that we won't know if people are using certain functions from a report or not? if that's the case, it's fine.

Reply:
Removed it. Also, added an exception handler to log failing functions.